packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -176,10 +177,12 @@ describe('API requests', () => {
       subscriptionId: 'sub_987675',
       planId: 'plan_2345',
       productId: 'prod_4567',
+      paymentProvider: 'paypal',
```

Comment:
`params` into `apiCancelSubscription` and `apiUpdateSubscriptionPlan` has a TS error: `Type 'string' is not assignable to type 'ProviderType | undefined'.ts(2345)`.

It looks like it wants me to declare the types `params` that I'm passing in for each test, since otherwise `paymentProvider` is implicitly of type `string`. I could do that, but it doesn't seem very DRY, since this is just for a test.

Can I just ignore it, or would you prefer I explicitly specify the types in `params`? Is there a more concise way than:
```ts
const params: {
  // keys: types
} = {
  // keys: values
}
```

Reply:
I think you could assert the type: `paymentProvider: 'paypal' as ProviderType`